src/main/java/com/google/devtools/build/lib/analysis/BuildView.java

```diff
@@ -410,7 +410,7 @@ public AnalysisResult update(
               "Analysis succeeded for only %d of %d top-level targets",
               numSuccessful, numTargetsToAnalyze);
       eventHandler.handle(Event.info(msg));
-      logger.atInfo().log(msg);
+      logger.atInfo().log("%s", msg);
```

Comment:
What's up with those changes? I don't see them in https://github.com/bazelbuild/bazel/commit/22c91dab43558503a1d6759ccfd77587d87b28cd

Reply:
There are two more (partial) cherry picks from: 1987af2216b3adb7d0e19a70f0e1778eee519d4c and ccc9b9d65623d454d987185af4e14ce26a67fa53 to fix the error prone failures.